src/include/wiredtiger.in

```diff
@@ -3696,13 +3696,15 @@ typedef enum {
 #define	WT_OPEN_CREATE		0x001
 /*! WT_FILE_SYSTEM::open_file flags: direct I/O requested */
 #define	WT_OPEN_DIRECTIO	0x002
+/*! WT_FILE_SYSTEM::open_file flags: durable creation required */
+#define	WT_OPEN_DURABLE		0x004
```

Comment:
Seeing this change made me realize the ``WT_OPEN_..`` isn't an adequate prefix. Would you mind changing this to ``WT_FS_OPEN_..``